src/GraphMetadataViewExtension/GraphMetadataView.xaml

```diff
@@ -72,68 +124,100 @@
                     Grid.Row="0">
             <GroupBox Header="Description"
                       Height="100">
-                <TextBox Text="{Binding GraphDescription, Mode=TwoWay, FallbackValue='Graph Description'}"/>
+                <TextBox Text="{Binding GraphDescription, Mode=TwoWay, FallbackValue='Graph Description'}" Style="{StaticResource TextBoxStyle}"/>
             </GroupBox>
             <GroupBox Header="Image">
                 <controls:ImageSelector MaxHeight="150"
                                         MinHeight="100"
                                         Image="{Binding Thumbnail, Mode=TwoWay}"/>
             </GroupBox>
             <GroupBox Header="Author Name">
-                <TextBox Text="{Binding GraphAuthor, Mode=TwoWay}" />
+                <TextBox Text="{Binding GraphAuthor, Mode=TwoWay}" Style="{StaticResource TextBoxStyle}" />
             </GroupBox>
-            <GroupBox Header="Learn more URL">
-                <TextBox Text="{Binding HelpLink, Converter={StaticResource UriToStringConverter}, Mode=TwoWay}" />
-            </GroupBox>
-            <Separator />
+            <StackPanel>
+                <StackPanel.Resources>
+                    <ControlTemplate x:Key="errorTemplate">
+                        <StackPanel Orientation="Vertical">
+                            <AdornedElementPlaceholder/>
+                            <TextBlock FontStyle="Italic"
+                                       Foreground="Red"
+                                       HorizontalAlignment="Left"
+                                       Margin="4"
+                                       Text="{Binding Path=ErrorContent}"/>
+                        </StackPanel>
+                    </ControlTemplate>
+                </StackPanel.Resources>
+
+                <GroupBox Header="Learn more URL" Margin="10px,5px, 10px, 0px">
+                    <TextBox Margin="4" Validation.ErrorTemplate="{StaticResource ResourceKey=errorTemplate}"
```

Comment:
NIT: I think having each parameter on a separate line makes it a lot easier to read